pandas/sparse/tests/test_frame.py

```diff
@@ -768,12 +768,20 @@ def _check(frame, orig):
         self._check_all(_check)
 
     def test_count(self):
-        result = self.frame.count()
         dense_result = self.frame.to_dense().count()
+
```

Comment:
can you also (maybe in another function), run thru all stat methods (you can just assert that they don't raise), e.g. just do a loop for all named (stat functions).